src/components/calculator/util/__tests__/FormatPrecision.test.ts

```diff
@@ -22,6 +29,9 @@ describe('fixedPointPrecision', () => {
   it('shows precision for numbers close to 1e6', () => {
     expect(fixedPointPrecision(996700)).toEqual('997,000')
     expect(fixedPointPrecision(999920)).toEqual('999,920')
+    // This isn't super right, but it also seems odd to report 100% when it's not?
+    expect(fixedPointPrecision(1e6 - 0.1)).toEqual('999,999')
```

Comment:
If we round 99 to 100 why wouldn't we round this to 1,000,000?

Reply:
We round 99 to 100, but we _don't_ round 99% to 100%
expect(fixedPointPrecisionPercent(0.9992)).toEqual('99.92%')

Reply:
Oh I see

Reply:
Come to think of it, we should actually "round" 1e6 to 999,999 - because I never want to say something is certain.